CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v0.10.0 (Released 2025-03-04)
+
+BREAKING CHANGES
+
+- transfers: switch to newer `/accounts/{accountID}`-prefixed endpoints
+- transfers: update transfer methods to accept an accountID parameter
```

Comment:
think we can remove this line since that's reflected in the line above already

Reply:
done